src/input_xml.F90

```diff
@@ -2188,9 +2188,11 @@ subroutine read_tallies_xml()
            call get_node_value(node_mesh, "type", temp_str)
       select case (to_lower(temp_str))
       case ('rect', 'rectangle', 'rectangular')
-        m % type = LATTICE_RECT
-      case ('hex', 'hexagon', 'hexagonal')
-        m % type = LATTICE_HEX
+        call warning("Mesh type '" // trim(temp_str) // "' is deprecated. &
```

Comment:
In addition, the "python/lattice/nested", "python/lattice/simple" and "python/pincell" examples fail since the Python API no longer accepts the "regular" mesh option.